BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -140,7 +140,10 @@ public float getSWLWOffset
         public bool isTimed = false;
 
         [KSPField]
-        public bool radarLOAL = false;
+        public bool radarLOAL = false;                              //if true, radar missile will acquire and lock onto a target after launch, using the missile's onboard radar
+
+        [KSPField]
+        public bool canRelock = true;                               //if true, if a FCS radar guiding a SARH missile loses lock, the missile will be switched to the active radar lock instead of going inactive from target loss.
 
         [KSPField]
         public bool canRelock = true;
```

Comment:
Looks like something went slightly wrong with the diff, the `canRelock` field is duplicated.